src/audiovideocontroller/DefaultAudioVideoController.ts

```diff
@@ -242,6 +244,166 @@ export default class DefaultAudioVideoController
       this._webSocketAdapter,
       this.logger
     );
+  }
+
+  private prestart(): void {
+    this.logger.info('Pre-connecting signaling connection.');
+    this.createOrReuseSignalingTask()
+      .run()
+      .catch(e => {
+        this.logger.error(`Signaling task pre-start failed: ${e}`);
+
+        // Clean up just in case a subsequent attempt will succeed.
+        this.signalingTask = undefined;
+      });
+  }
+
+  start(options?: { signalingOnly?: boolean }): void {
+    this.startReturningPromise(options)
+      .then(() => {
+        this.logger.info('start completed');
+      })
+      // Just-in-case error handling.
+      .catch(
+        /* istanbul ignore next */
+        e => {
+          this.logger.error(`start failed: ${e}`);
+        }
+      );
+  }
+
+  // For tests only.
```

Comment:
nit: Is this accurate since we called it under start()?

Reply:
The comment implies "this is public for tests only". I'll clarify 'cos evidently the implication isn't clear enough!